endaq/ide/info.py

```diff
@@ -319,10 +319,10 @@ def to_pandas(
         t = t + np.datetime64(channel.dataset.lastUtcTime, "s")
     elif time_mode != "timedelta":
         raise ValueError(f'invalid time mode "{time_mode}"')
+    
+    if hasattr(channel, "subchannels"):
+        columns = [sch.name for sch in channel.subchannels]
+    else:
+        columns = [channel.name]
 
-    result = pd.DataFrame(
-        data, index=t, columns=[sch.name for sch in channel.subchannels]
-    )
-    result.index.name = "timestamp"
```

Comment:
Should this be pluralized?   I'm not saying it should, just asking

Reply:
Which, `channel.subchannels`? That's correct.

Reply:
I was referring to the column title "timestamp"

Reply:
Actually as I say out loud the other titles, they would be not pluralized.    So I think the answer to my original question would be 'no' haha